nondeterministic test fail in TrialTestPayjoin4.test_low_feerate

There is a somewhat rare, nondeterministic test failure in `jmclient/test/test_payjoin.py:TrialTestPayjoin4.test_low_feerate`

stdout:
```
User data location: .
2021-08-05 15:30:29,969 [INFO]  Detected new wallet, performing initial import
2021-08-05 15:30:33,740 [INFO]  Detected new wallet, performing initial import
2021-08-05 15:30:34,138 [INFO]  Using this manually set tx feerate (randomized for privacy): 1449 sat/vkB (1.4 sat/vB).
2021-08-05 15:30:34,141 [INFO]  Using this manually set tx feerate (randomized for privacy): 1101 sat/vkB (1.1 sat/vB).
2021-08-05 15:30:34,148 [INFO]  Using a fee of: 0.00000154 BTC (154 sat).
2021-08-05 15:30:34,149 [INFO]  Using a change value of: 0.89999846 BTC (89999846 sat).
Completed PSBT created: 
{
    "psbt-version": 0,
    "unsigned-tx": {
        "hex": "020000000132f14fc4851a10accab8bc804bd7cd61ee0d1b7d15c09d81b9209cf81bea35e90000000000fdffffff0280778e060000000016001461c8c61466c79be5de002bdcf71409ac15dfa09de6495d0500000000160014f6fa0accf1f9c086b208540fa8e833da4b33a9fa7f0b0000",
        "inputs": [
            {
                "outpoint": "e935ea1bf89c20b9819dc0157d1b0dee61cdd74b80bcb8caac101a85c44ff132:0",
                "scriptSig": "",
                "nSequence": 4294967293
            }
        ],
        "outputs": [
            {
                "value_sats": 110000000,
                "scriptPubKey": "001461c8c61466c79be5de002bdcf71409ac15dfa09d",
                "address": "bcrt1qv8yvv9rxc7d7thsq90w0w9qf4s2algyana76t4"
            },
            {
                "value_sats": 89999846,
                "scriptPubKey": "0014f6fa0accf1f9c086b208540fa8e833da4b33a9fa",
                "address": "bcrt1q7maq4n83l8qgdvsg2s8636pnmf9n8206va75us"
            }
        ],
        "txid": "158f9f648c7c9391c594ade8cd496c6d5daad1c67347dc6d1208d0c2cada94cf",
        "nLockTime": 2943,
        "nVersion": 2
    },
    "psbt-inputs": [
        {
            "input-index": 0,
            "utxo": {
                "value_sats": 200000000,
                "scriptPubKey": "0014783900c7054679ec483f1bb4dad878c5c9f0db77",
                "address": "bcrt1q0qusp3c9geu7cjplrw6d4krcchylpkmhfhjd2v"
            },
            "final-scriptSig": "",
            "final-scriptWitness": "0247304402203e92d1f733f9416bcfb2fdd3031d5edb83e3e14f08df4d8f1b8e4e5e60d48af202205eb0b28546d0e9d5766e2e5182549e684fa763380c9eb0c01076e5bb2643580101210317a96cac3ba2853a3e7c0dcb51dd08b8be9ef579cebddf3029b1b7b83c120dac"
        }
    ],
    "psbt-outputs": [
        {
            "output-index": 0
        },
        {
            "output-index": 1
        }
    ]
}
Choosing one coin at random
We selected inputs worth: 200000000
Failed: r, c:  b'<html><head><title>Processing Failed</title></head><body><b>Processing Failed</b></body></html>' 500
```

stacktrace:
```
Traceback (most recent call last):
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/http.py", line 1755, in dataReceived
    finishCallback(data[contentLength:])
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/http.py", line 2171, in _finishRequestBody
    self.allContentReceived()
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/http.py", line 2284, in allContentReceived
    req.requestReceived(command, path, version)
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/http.py", line 946, in requestReceived
    self.process()
--- <exception caught here> ---
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/server.py", line 235, in process
    self.render(resrc)
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/server.py", line 302, in render
    body = resrc.render(self)
  File "joinmarket-clientserver/jmvenv/lib/python3.8/site-packages/twisted/web/resource.py", line 265, in render
    return m(request)
  File "joinmarket-clientserver/jmclient/test/test_payjoin.py", line 48, in render_POST
    assert retval[0]
builtins.AssertionError: assert False


jmclient/test/test_payjoin.py:150 (TrialTestPayjoin4.test_low_feerate)
result = None, g = <generator object _runSequentially at 0x7f939a5cb3c0>
status = _CancellationStatus(deferred=<Deferred at 0x7f939a6b05e0 current result: None>, waitingOn=<Deferred at 0x7f939a612640 current result: None>)

    @failure._extraneous
    def _inlineCallbacks(result, g, status):
        """
        Carry out the work of L{inlineCallbacks}.
    
        Iterate the generator produced by an C{@}L{inlineCallbacks}-decorated
        function, C{g}, C{send()}ing it the results of each value C{yield}ed by
        that generator, until a L{Deferred} is yielded, at which point a callback
        is added to that L{Deferred} to call this function again.
    
        @param result: The last result seen by this generator.  Note that this is
            never a L{Deferred} - by the time this function is invoked, the
            L{Deferred} has been called back and this will be a particular result
            at a point in its callback chain.
    
        @param g: a generator object returned by calling a function or method
            decorated with C{@}L{inlineCallbacks}
    
        @param status: a L{_CancellationStatus} tracking the current status of C{g}
        """
        # This function is complicated by the need to prevent unbounded recursion
        # arising from repeatedly yielding immediately ready deferreds.  This while
        # loop and the waiting variable solve that by manually unfolding the
        # recursion.
    
        waiting = [True, # waiting for result?
                   None] # result
    
        while 1:
            try:
                # Send the last result back as the result of the yield expression.
                isFailure = isinstance(result, failure.Failure)
                if isFailure:
                    result = result.throwExceptionIntoGenerator(g)
                else:
>                   result = g.send(result)

../jmvenv/lib/python3.8/site-packages/twisted/internet/defer.py:1418: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

callables = [<function TestCase._runCleanups.<locals>._makeFunction.<locals>.<lambda> at 0x7f939a5944c0>]
stopOnFirstError = False

    @defer.inlineCallbacks
    def _runSequentially(callables, stopOnFirstError=False):
        """
        Run the given callables one after the other. If a callable returns a
        Deferred, wait until it has finished before running the next callable.
    
        @param callables: An iterable of callables that take no parameters.
    
        @param stopOnFirstError: If True, then stop running callables as soon as
            one raises an exception or fires an errback. False by default.
    
        @return: A L{Deferred} that fires a list of C{(flag, value)} tuples. Each
            tuple will be either C{(SUCCESS, <return value>)} or C{(FAILURE,
            <Failure>)}.
        """
        results = []
        for f in callables:
            d = defer.maybeDeferred(f)
            try:
                thing = yield d
                results.append((defer.SUCCESS, thing))
            except Exception:
                results.append((defer.FAILURE, Failure()))
                if stopOnFirstError:
                    break
>       defer.returnValue(results)

../jmvenv/lib/python3.8/site-packages/twisted/trial/util.py:282: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

val = [(True, None)]

    def returnValue(val):
        """
        Return val from a L{inlineCallbacks} generator.
    
        Note: this is currently implemented by raising an exception
        derived from L{BaseException}.  You might want to change any
        'except:' clauses to an 'except Exception:' clause so as not to
        catch this exception.
    
        Also: while this function currently will work when called from
        within arbitrary functions called from within the generator, do
        not rely upon this behavior.
        """
>       raise _DefGen_Return(val)
E       twisted.internet.defer._DefGen_Return: [(True, None)]

../jmvenv/lib/python3.8/site-packages/twisted/internet/defer.py:1362: _DefGen_Return

During handling of the above exception, another exception occurred:

self = <test_payjoin.TrialTestPayjoin4 testMethod=test_low_feerate>

    def tearDown(self):
        for dc in reactor.getDelayedCalls():
            dc.cancel()
        res = final_checks(self.wallet_services, self.cj_amount,
>                          self.manager.final_psbt.get_fee(),
                           self.ssb, self.rsb)
E       AttributeError: 'NoneType' object has no attribute 'get_fee'

test/test_payjoin.py:129: AttributeError
```

--- a/jmclient/test/test_payjoin.py
+++ b/jmclient/test/test_payjoin.py
@@ -150,9 +150,12 @@ class TrialTestPayjoin4(PayjoinTestBase, unittest.TestCase):
         jm_single().config.set("POLICY", "txfees", self.old_txfees)
     def test_low_feerate(self):
         self.old_txfees = jm_single().config.get("POLICY", "tx_fees")
-        # set such that randomization cannot pull it below minfeerate
-        # (default of 1.1 sat/vbyte):
-        jm_single().config.set("POLICY", "tx_fees", "1376")
+        # To set such that randomization cannot pull it below minfeerate
+        # (default of 1.1 sat/vbyte), we should need tx_fees at 1376
+        # (which corresponds to 1.108); but given the minor potential deviations
+        # as noted in https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/babad1963992965e933924b6c306ad9da89989e0/jmclient/jmclient/payjoin.py#L802-L809
+        # , we increase from that by 2%.
+        jm_single().config.set("POLICY", "tx_fees", "1404")
         d = self.do_test_payment(SegwitWallet, SegwitWallet)
         d.addCallback(self.reset_fee)
         return d

